core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api/Access.kt

```diff
@@ -706,6 +706,41 @@ class Access : TestBase() {
         // SampleEnd
     }
 
+    @Test
+    fun columnSelectors_kproperties() {
+        // SampleStart
+        // by column name
+        df.select { it[Person::name] }
+        df.select { (Person::name)() }
+        df.select { col(Person::name) }
+
+        // by column path
+        df.select { it[Person::name][Name::firstName] }
+        df.select { Person::name[Name::firstName] }
+
+        // with a new name
+        df.select { Person::name named "Full Name" }
+
+        // converted
+        df.select { Person::name[Name::firstName].map { it.lowercase() } }
+
+        // column arithmetics
+        df.select { 2021 - (Person::age)() }
+
+        // two columns
+        df.select { Person::name and Person::age }
+
+        // range of columns
+        df.select { Person::name..Person::age }
```

Comment:
Very strange API:)

Reply:
not weirder than "name".."age" imo

Reply:
Of course not, I mean totally the range of columns based on name, I could imagine ranging the ordered things or alphabetically, but not the range of column names in the structure, is this order guaranteed by compiler or somehow?

Reply:
Nope, that's impossible afaik. However, it's still very useful to select multiple columns quickly. But, as with all non-typesafe operations (resulting in `Any?` types), it works best in Jupyter or with the upcoming plugin, which generate accessors on the fly.